js_modules/dagster-ui/packages/ui-core/src/graph/OpTags.tsx

```diff
@@ -377,6 +420,134 @@ export const KNOWN_TAGS = {
     icon: polars,
     content: 'Polars',
   },
+  catboost: {
+    color: null,
+    icon: catboost,
+    content: 'CatBoost',
+  },
+  rust: {
+    color: '#000000',
+    icon: rust,
+    content: 'Rust',
+    reversed: true,
+  },
+  pytorchlightning: {
+    color: null,
+    icon: pytorch_lightning,
+    content: 'Pytorch Lightning',
+  },
+  deltalake: {
+    color: '#00ADD4',
+    icon: delta_lake,
+    content: 'Delta Lake',
+  },
+  parquet: {
+    color: '#50ABF1',
+    icon: parquet,
+    content: 'Parquet',
+  },
+  lightgbm: {
+    color: null,
+    icon: lightgbm,
+    content: 'lightgbm',
+  },
+  xgboost: {
+    color: '#1A9EDB',
+    icon: xgboost,
+    content: 'xgboost',
```

Comment:
"XGBoost"?

Reply:
this was specifically requested by a user